docs/quickstarts/README.md

```diff
@@ -115,6 +115,40 @@ If you have created quick starts in the Hybrid Cloud Console before, start here:
 
     e. Get your quick start reviewed by stakeholders as needed.
 
+6. On your issue branch, add new or changed quick start files to stage.
+
+    ```
+     $[issue-branch] git add -A 
```

Comment:
It would be better to instruct the user to only add the changed files, in case something else snuck in. (-A is 'all')
```suggestion
     git add <updated files> 
```

Reply:
That's fine.